ai-ml/trainium-inferentia/eks.tf

```diff
@@ -116,12 +116,14 @@ module "eks" {
       max_size     = 8
       desired_size = 3
 
-      instance_types = ["m5.xlarge"]
+      instance_types = ["m5.2xlarge"]
 
       labels = {
-        WorkerType    = "ON_DEMAND"
-        NodeGroupType = "core"
-        workload      = "rayhead"
+        WorkerType      = "ON_DEMAND"
+        NodeGroupType   = "core"
+        workload        = "rayhead"
+        instanceType    = "mixed-x86"
+        provisionerType = "Karpenter"
```

Comment:
These labels are for  Karpenter to spinup the nodes, but not the Managed Node groups with CA. I would say remove these or update as below.

```tf
        provisionerType = "ClusterAutosclaer"
```

If you want to deploy Llama2 model on Managed Nodegroups with these instances then you have to update the Ray deployment yaml with unique label that is used only by this node group

Reply:
Ok , addressed